core/validator.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_date(value):
+    if value > datetime.date.today():
+        raise ValidationError(
+            _('%(value)s invalid purchase date'), params={'value': value},
```

Comment:
As an admin it might to be clear on the onset as to what makes a purchase date invalid. Update this message to be more convey the meaning. Thanks.

Reply:
Can you suggest? Or is this okay `Purchase date cannot be in the future`

Reply:
That sounds better @QUDUSKUNLE. It does leave out one crucial part though; what a valid purchase date it. There's no harm in being verbose as necessary.

Reply:
> "2018-07-02 is invalid. Purchase date can only be set to today or a date in the past"